packages/app/src/index.ts

```diff
@@ -0,0 +1,11 @@
+
+import { AeroGearConfiguration, coreInstance } from "@aerogear/core";
+
+/**
+ * Initialize all platform SDK's by using configuration generated by AeroGear command line client
+ *
+ * @param config object containing SDK. Object can be read from file or pasted directly in source code.
+ */
+export const init = (config: AeroGearConfiguration): void => {
+  return coreInstance.init(config);
```

Comment:
`void` seems wrong here, isn't it `Promise<void>`?

Reply:
This has huge implication for users that need to move to async api. It makes it really cumbersome to work with angular. This api do not require anything that is async as it is just setting up configuration.